.github/workflows/release.yml

```diff
@@ -1,4 +1,4 @@
-name: changeset
+name: Release
 
 on:
   workflow_dispatch:
```

Comment:
There's a workflow dispatch option on this - should we drop it given you have the manual workflow?

Reply:
Yep, no need for it here